src/plugins/controls/public/services/options_list/types.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-import { OptionsListRequest, OptionsListResponse } from '../options_list/types';
+import { OptionsListRequest, OptionsListResponse } from '../../../common/options_list/types';
```

Comment:
Did we come to a consensus about exporting all types from the root `types` file in `common`, `public`, and `server`? I think it'd look nicer if we imported from `../../../common/types` because I prefer shorter import paths, but this isn't a dealbreaker.

Reply:
Hmmm... I honestly don't remember if we decided that, haha! :) I looked in to it and it seems like this would require a lottt of import changes if we wanted to be consistent with it - for example, `src/plugins/controls/public/control_group/types.ts` is exporting stuff from `../../common/control_group/types` and so I would have to go through and change all of the imports of `.../public/control_group/types.ts` to use the `common` import instead.

Totally cool doing that, but I don't think it makes sense to make all those changes as part of this PR - maybe I could make a follow up if Devon agrees that shorter import paths is better? I honestly don't have a strong preference so I'm good with either 😆

Reply:
I think that within a plugin having longer import paths is fine. If we're in a file related to `options_list` in public and we have to import from `../../../common/options_list/types` I think that works great because it ties the disparate parts of the plugin together nicely.